### ISSUE TITLE
Rework donation feature

### ISSUE THREAD

**Issue by lsolesen** [vih-elevforening/elevforeningen-build#14]
Right now this is the initial screen when preparing to attend the yearly student meeting.

![remove get membership](https://cloud.githubusercontent.com/assets/148026/6709984/ce40d22e-cd7d-11e4-9c4a-ec286884c4cf.png)

However, we changed or strategy for the funding. This means that they should just be faced with the option to attend the meeting.

On the next pane however, we should be able to create an upsell page, where we can convince the attendee that it is a good idea to be a part of the fund. The user should be able to add it's SSN number (which could be saved as a hidden field on the user - and not on the line item - off course it could be saved there also).  Adding the SSN number is voluntary (only add it if you want a tax reduction for your donation). 

So basically we want everybody to see the pane in the future, but not until they put in their information to join the meeting - and we want to be able to edit some basic stuff like advantages etc. It could all be handled by a pane if that is the best way to do it.

**Comment by mglaman** [comment]
@lsolesen So the CPR number will be hidden on the Paid Event add to cart form. Currently the code checks "if a CPR number exists, display the donation form." Now it will display the donation form always - and the CPR number is just for the individual to receive a tax reduction, correct?

**Comment by mglaman** [comment]
The Donation pane has a title and description text area - will these be suitable?

Sounds like we just need to expose the CPR field there. Should it be moved to an order field and set it that way? Or just to user?

**Comment by lsolesen** [comment]
It is really important that the CPR-number is never displayed to anyone but admin people. This is highly confident information. I think the smartest thing would be to save it on the user, as then it would be available the next time also. However, it might be easier to just save it on the order?

**Comment by mglaman** [comment]
Cool, the field can be added to the user and hidden from display. Did you need it added anywhere specific?

**Comment by lsolesen** [comment]
I just need to be able to pull it out to give the tax authorities :)
Den 02/06/2015 17.37 skrev "Matt Glaman" <notifications@github.com>:
